modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -206,6 +207,43 @@ function acquia_cms_site_studio_modules_installed($modules, $is_syncing) {
         'administer style_guide',
       ]);
     }
+    if (in_array('node_revision_delete', $modules)) {
+      // Set default configurations for node revision delete module.
+      $config = \Drupal::configFactory()->getEditable('node_revision_delete.settings');
+      if ($config) {
+        $config->set('defaults.amount.status', TRUE)
+          ->set('defaults.amount.settings.amount', 50)
+          ->set('defaults.created.status', TRUE)
+          ->set('defaults.created.settings.age', 12)
+          ->set('defaults.drafts.status', TRUE)
+          ->set('defaults.drafts.settings.age', 12)
+          ->save();
+      }
+
+      // Set default node revision delete configuration for Page content type.
+      if (\Drupal::service('module_handler')->moduleExists('acquia_cms_page')) {
+        $type = NodeType::load('page');
+        $third_party_settings = $type ? $type->get('third_party_settings') : NULL;
+        if ($type && !isset($third_party_settings['node_revision_delete'])) {
+          $type->setThirdPartySetting('node_revision_delete', 'amount', [
+            'status' => TRUE,
+            'settings' => [
+              'amount' => 30,
+            ],
+          ]);
+          foreach (['created', 'drafts', 'drafts_only'] as $value) {
+            $type->setThirdPartySetting('node_revision_delete', $value, [
+              'status' => FALSE,
+              'settings' => [
+                'age' => 0,
+              ],
+            ]);
+          }
+
+          $type->save();
+        }
+      }
+    }
```

Comment:
Is this the default configuration value provided by Node Revision Delete, or is it a standard provided by ACMS?

Reply:
These  node revision delete configuration are overridden and shipped by ACMS